demokratie-live/democracy-client
# Sprint#7/fix permissions

**Issue:** #172

**Description:**
Removed unnessecary permissions

**How to test:**
- Deploy to Playstore
- Check if Permission Dialog no longer appears
- Check if App works properly
(works fine in dev mode)

**Checklist:**
